elastic/datadog_checks/elastic/data/conf.yaml.example

```diff
@@ -182,6 +182,13 @@ instances:
     #
     # custom_queries: []
 
+    ## @param submit_events - boolean - optional - default: true
+    ## By default we submit events to Datadog if the cluster starts unhealthy or anytime its health changes.
+    ## Some customers find this too noisy and prefer to define their own custom alerts based on metrics only.
+    ## For those cases submitting events can be disabled by setting this option to 'false'.
+    #
+    # submit_events: true
```

Comment:
nit - this is probably a very personal opinion but I would not write 'Some customers <whatever>" in a public doc, which is also shipped with the agent and which is going to be read by customers. I would write something more neutral such as:

```
## By default we submit events to Datadog if the cluster starts unhealthy or anytime its health changes.
## You can set this option to `false` to disable submitting the events in case you already implemented your own custom alerts based on metrics.
```